cineast-core/src/main/java/org/vitrivr/cineast/core/data/TemporalObject.java

```diff
@@ -0,0 +1,44 @@
+package org.vitrivr.cineast.core.data;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Comparator;
+import java.util.List;
+
+public class TemporalObject {
+
+  private final List<String> segments;
+  private final String objectId;
+  private final double score;
+
+  @JsonCreator
+  public TemporalObject(@JsonProperty("segments") List<String> segments, @JsonProperty("objectId") String objectId, @JsonProperty("score") double score) {
+    this.segments = segments;
+    this.objectId = objectId;
+    this.score = score;
+  }
+
+  public String getObjectId() {
+    return objectId;
+  }
+
+  public double getScore() {
+    return score;
+  }
+
+  public List<String> getSegments() {
+    return segments;
+  }
+
+  @Override
+  public String toString() {
+    return "TemporalObject(" + objectId + ", " + score + ")";
+  }
+
+  public static final Comparator<TemporalObject> COMPARATOR = new Comparator<TemporalObject>() {
+    public int compare(TemporalObject o1, TemporalObject o2) {
+      return Double.compare(o2.getScore(), o1.getScore());
```

Comment:
It's not obvious to me why this is not `Double.compare(o1.getScore(), o2.getScore())`, can you add some documentation here? Especially since Comparators should behave as expected, otherwise you could maybe call the variable `INVERSE_COMPARATOR`?

Reply:
That was a mistake from my side. Is fixed with the following commit.